crates/bevy_render/src/render_ecs_resource.rs

```diff
@@ -0,0 +1,44 @@
+use std::marker::PhantomData;
+
+use bevy_app::{App, Plugin};
+use bevy_ecs::system::{
+    lifetimeless::{SCommands, SRes},
+    Resource, RunSystem, SystemParamItem,
+};
+
+use crate::{RenderApp, RenderStage};
+
+/// This plugin extracts the resources into the "render world".
+///
+/// Therefore it sets up the [`RenderStage::Extract`](crate::RenderStage::Extract) step
+/// for the specified [`Resource`].
+pub struct ExtractResourcePlugin<R>(PhantomData<R>);
+
+impl<R> Default for ExtractResourcePlugin<R> {
```

Comment:
All other uses of `R` have this bound, so it should be probebly be used here too.
```suggestion
pub struct ExtractResourcePlugin<R: Clone + Resource>(PhantomData<R>);

impl<R: Clone + Resource> Default for ExtractResourcePlugin<R> {
```